VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimStatus.tsx

```diff
@@ -143,7 +143,7 @@ function ClaimStatus({ claim, claimType }: ClaimStatusProps) {
     <Box {...testIdProps('Your-claim: Status-tab-claim-details-page')} testID="claimStatusDetailsID">
       {renderActiveClaimStatusDetails()}
       {renderClosedClaimStatusDetails()}
-      <NeedHelpData claimId={claim.id} claimType={claim.attributes.claimType} claimPhase={claim.attributes.phase} />
+      <NeedHelpData />
```

Comment:
Are we no longer tracking the analytic for this?

Reply:
See above